bitnami/rabbitmq/templates/secrets.yaml

```diff
@@ -1,4 +1,18 @@
-{{- if or (not .Values.auth.existingErlangSecret) (not .Values.auth.existingPasswordSecret) }}
+{{- if not (and .Values.auth.existingErlangSecret .Values.auth.existingPasswordSecret) }}
+{{- $rabbitmq_password := randAlphaNum 16 | b64enc | quote }}
+{{- $rabbitmq_erlangCookie := randAlphaNum 32 | b64enc | quote }}
+{{- $secret := (lookup "v1" "Secret" .Release.Namespace (include "rabbitmq.fullname" .)) }}
+{{- if $secret }}
+  {{- $rabbitmq_password := index $secret.data "rabbitmq-password" }}
+  {{- $rabbitmq_erlangCookie := index $secret.data "rabbitmq-erlang-cookie" }}
+{{- end -}}
+{{- if .Values.auth.password }}
+   {{ $rabbitmq_password := .Values.auth.password }}
+{{- end -}}
+{{- if .Values.auth.erlangCookie }}
+   {{ $rabbitmq_erlangCookie := .Values.auth.erlangCookie }}
+{{- end -}}
+
```

Comment:
We actually have a common method implemented for this same purpose:

https://github.com/bitnami/charts/blob/14d91e2bed2e4328434c45f301628987f53e43ec/bitnami/common/templates/_secrets.tpl#L85

You can take a look at this PR, which implements a similar change:
https://github.com/bitnami/charts/pull/9300/files

Reply:
Thansk for the tip, I'm gonna take a look. That ought to make this PR much simpler :+1:

Reply:
So I rewrited the PR to basically do the same thing that the linked one, opinions ?

Reply:
Great!! It is much simpler now, thanks 😁